### PR TITLE
rename all SidechainDB class according to bitcoin style guide

### PR DESCRIPTION
**Pull request by joemphilips**
#9

### CODE REVIEW

**Comment by CryptAxe** [comment]
Almost everything is perfect, just a few changes to be made that I'll point out :+1:

File: src/sidechaindb.h [hunk 1]
```diff
@@ -2,8 +2,8 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#ifndef BITCOIN_SIDECHAINDB_H
```
Comment (CryptAxe) [inline src/sidechaindb.h]: These statements at the top of headers in C++ (called include guards) are supposed to be the filename, so this actually should stay as BITCOIN_SIDECHAINDB_H. It's a style decision how these are actually named, but to stay in line with Bitcoin these should remain BITCOIN_SIDECHAINDB_H.

File: src/sidechaindb.h [hunk 2]
```diff
@@ -109,5 +109,5 @@ class SidechainDB
     bool ApplyDefaultUpdate();
 };
 
-#endif // BITCOIN_SIDECHAINDB_H
```
Comment (CryptAxe) [inline src/sidechaindb.h]: Should also remain as BITCOIN_SIDECHAINDB_H here.

**Comment by CryptAxe** [comment]
Alright so just undo renaming the include guards (top and bottom of the sidechaindb.h file) and push that commit into your branch and everything can be squashed / merged. Thanks again! :+1:

**Comment by joemphilips** [comment]
seems all right now? :)
I wonder why textbook for C++ is so thick... :(

**Review by CryptAxe — APPROVED** [review]

**Pull request merged**